src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -330,6 +373,17 @@ typedef struct uct_ib_mlx5_md {
 
     /* Cached values of counter set id per port */
     uint8_t                  port_counter_set_ids[UCT_IB_DEV_MAX_PORTS];
+
+    /* CQ for indirect (UMR) mkeys */
+    struct ibv_cq            *umr_cq;
+    /* QP for indirect (UMR) mkeys */
+    struct ibv_qp            *umr_qp;
+
+    /* Indirect (UMR) mkey pool (on the host) */
+    ucs_list_link_t          umr_mkey_pool;
+
+    /* Hash map of indirect mkey (from the host) to mkey alias (on the DPU) */
+    khash_t(umr_mkey_map)    *umr_mkey_hash;
```

Comment:
maybe put all umr stuff under sub-struct "umr" ?

Reply:
Ok, I've put it like:
```
    struct {
        /* CQ for indirect (UMR) mkeys */
        struct ibv_cq            *cq;
        /* QP for indirect (UMR) mkeys */
        struct ibv_qp            *qp;
        /* Indirect (UMR) mkey pool (on the host) */
        ucs_list_link_t          mkey_pool;
        /* Hash map of indirect mkey (from the host) to mkey alias (on the DPU) */
        khash_t(umr_mkey_map)    *mkey_hash;
    } umr;